streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -517,9 +526,9 @@ void handleRevocation(final Collection<TopicPartition> revokedPartitions) {
         }
 
         if (!remainingRevokedPartitions.isEmpty()) {
-            log.warn("The following partitions {} are missing from the task partitions. It could potentially " +
+            log.debug("The following revoked partitions {} are missing from the current task partitions. It could potentially " +
```

Comment:
Making this debug since `warn` seems too intense, and I'm not sure it's even worthy of `info` -- also, with named topologies you would expect to see this almost every time a topology is removed